### ISSUE TITLE
Suggestion: Allowing maps to store data for "name" and "author"

### ISSUE THREAD

**Issue by Rushnett** [Pocketkid2/Survival-Games#2]
This is merely a suggestion. I believe after arena creation, there should be an option to assign an arena with the value of a name and author, where it is then displayed right before the countdown to the game starts. Maybe even replace the last line of the join sign to the map name? (http://media.rushnett.com/s/2015-09-30_0456.png)

Example: http://media.rushnett.com/s/2015-09-30_0454.png

**Comment by Pocketkid2** [comment]
This is an interesting idea. Prior to my work on this project, I had thought it would be more interesting to write my own SG plugin, mirroring the system and usage of this one (with better performance, better stability, better configurability, etc...). One of the features I had intented to implement was making games/arenas store ID's by name (a string value) instead of an integer number value. I'm sure something similar could be implemented, although that would be most likely tied to the arena itself, making things challenging. 

See, the way you suggest, if people want to have like 5 games with the same arena they have to set those values over and over again. Currently arenas aren't modular, making it hard to prevent the need to copy. I'm sure that's fine for most, but I'd like to hear what you think on the issue.

**Comment by Rushnett** [comment]
Overall, it would be best to rewrite the arena creation system, and make it a bit more dynamic and less of a hassle later on (ex wanting to delete arena 1, swapping arena 1 with 2, renaming, etc)

**Comment by Pocketkid2** [comment]
Yeah :) It might take a lot of planning and design to make it work best
